using/rendering-questions.md

```diff
@@ -2,59 +2,238 @@
 
 The PIE [Packaging Tool](packaging-questions.md) is used to assemble and package the Javascript and HTML into files needed to render an Assessment Item in the browser.
 
-An Assessment Item packed by the Packaging Tool may have some or all of the following files:
+The command uses 2 input files: 
```

Comment:
I have a separate page - packaging.md that explains the packaging process, this page is meant for ppl interested in rendering (ppl who may have no idea about packaging )